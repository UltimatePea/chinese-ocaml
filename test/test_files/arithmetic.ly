「：骆言测试程序：基本算术：」

设「甲」为１０
设「乙」为５
设「和」为「甲」＋「乙」
设「差」为「甲」－「乙」
设「积」为「甲」＊「乙」
设「商」为「甲」／「乙」
设「余」为「甲」％「乙」

打印 『和： 』
打印 「和」
打印 『差： 』
打印 「差」
打印 『积： 』
打印 「积」
打印 『商： 』
打印 「商」
打印 『余数： 』
打印 「余」