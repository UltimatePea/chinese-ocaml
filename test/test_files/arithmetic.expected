和: 
15
差: 
5
积: 
50
商: 
2
余数: 
0